dist/templates/ovnkube-identity.yaml.j2

```diff
@@ -0,0 +1,140 @@
+# ovnkube-identity
+# starts ovnkube-identity
+# it is run on the master(s).
+kind: Deployment
```

Comment:
Yes :)
Note that it is only used to test the HA aspect of the approver since the API server is talking to the local webhook. In our kind deployments the API runs on one node only.